dbcon/joblist/joblistfactory.cpp

```diff
@@ -1246,6 +1248,7 @@ const JobStepVector doAggProject(const CalpontSelectExecutionPlan* csep, JobInfo
         const WindowFunctionColumn* wc = NULL;
         bool hasAggCols = false;
         bool hasWndCols = false;
+        bool hasFuncCols = false;
```

Comment:
It is better to rename the flag so it brings the specific semantics, e.g. hasFuncColsWithSingleArg